Jenkinsfile

```diff
@@ -2,34 +2,37 @@
 
 REPOSITORY = 'govuk-content-schemas'
 
+// second boolean parameter describes whether the app is using the central
+// buildProject script - if so it will report its success individually, so this
+// job does not need to wait for it.
 def dependentApplications = [
```

Comment:
Hopefully we can get rid of the second param again soon, once everybody has started using `govuk.buildProject()`.